tests/models/test_DurableOrchestrationClient.py

```diff
@@ -161,7 +161,7 @@ async def test_get_200_get_status_success(binding_string):
 
     result = await client.get_status(TEST_INSTANCE_ID)
     assert result is not None
-    assert result.runtime_status == "Completed"
+    assert result.runtime_status.name == "Completed"
```

Comment:
Is there a way to make this conversion automatic?

Reply:
I wish! But the Python runtime isn't clever enough to do this. An alternative here would be to write this as
`assert result.runtime_status is OrchestrationRuntimeStatus.Completed`, which I guess circumvents the need for a string comparison. I could also see why that would be preferred. Want me to make the change?